### ISSUE TITLE
Add a timebox

### ISSUE THREAD

**Issue by martinkearn** [martinkearn/Azure-pass-manager#18]
Add the ability to create a time-box for an event. don't allow users to claim codes outside the time-box. This will help with fraud.

**Comment by daltskin** [comment]
If the start/end dates are stored against the event name then this validation can be done at the api level and will then work for the bot with no rework.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by leestott** [comment]
Yes I always set the expiry date to the last day of the event so passess cannot be redeemed outside of the event period.

**Comment by martinkearn** [comment]
Lee, the expiry date should be the date that is given with the codes when you receive them. This feature is a timebox where the codes can be redeemed, they are different things.
